ifc/templates/party/fraternityPartyList.html

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Yes, I dont know how or where to render it. I assume it's in the party,views.py file, right?